CI/bench/evaluate.py

```diff
@@ -188,17 +187,9 @@ def main():
 
             for root, dirs, files in os.walk("./evaluate"):
                 for file in files:
-                    if file.endswith("results_ranked_valid.csv"):
-                        file_path = os.path.join("./evaluate", file)
-                        df1 = pd.read_csv(file_path, usecols=["time_train_s", "time_infer_s","rank"])
-
                     if file.startswith("AutoGluon") and file.endswith(".csv") and "pairwise" in root:
                         file_path = os.path.join(root, file)
-                        df2 = pd.read_csv(file_path, usecols=["framework", "Winrate"])
-                        df1 = df1.assign(Winrate=None, framework=None)
-                        df1["Winrate"] = df2["Winrate"]
-                        df1["framework"] = df2["framework"]
-                        df1 = df1.reindex(columns=["framework", "Winrate", "rank", "time_train_s", "time_infer_s"])
+                        df1 = pd.read_csv(file_path, usecols=["framework", "Winrate", "time_train_s", "time_infer_s","rank"])
```

Comment:
@Innixma I still keep this in case we have a request to remove some columns/files in the future.

Reply:
Sounds good